### PR TITLE
Updates some uniform descriptions

### PR DESCRIPTION
**Pull request by Ty-Omaha**
## What Does This PR Do
Updates some uniform descriptions and names to be more accurate to what they are

## Why It's Good For The Game
to avoid confusion since they aren't in the navy

:cl:
tweak: Small edit to some uniform descriptions of Blueshield, Rep, and Magistrate.
/ 🆑

### CODE REVIEW

**Comment by Carthusia** [comment]
I would still argue that they are Navy, it's just that their particular assignment has limited their authority and require further authorisation to make use of it beyond the parameters of their assignment.

They are the only people on the station the captain has zero authority over and who only take orders from CC/NT Supreme Court. They can't order the BS, they can't order the NTR, and they can't order the Magistrate around.

**Comment by Ty-Omaha** [comment]
Argue all you want, but they aren't. It even explicitly says so in the NTR mission briefing.

> As a civilian, you should consider yourself an advisor, diplomat and intermediary.

**Comment by Sihsse** [comment]
> I would still argue that they are Navy

Setting aside the lore aspects you're arguing, we're more concerned with the game aspects as well as the implications of such that occur with players.

Players even associating them as Naval Personnel is a problem, especially newer players, since Central Command Naval Officers wield absolute authority over everyone on station, including but not limited to ignoring, changing, altering, and dictating Space Law as well as Standard Operating Procedure.

If a player confuses one of the VIP as a Naval Officer, all it takes is the player playing the specific role to decide not to correct them and flout authority that they don't have, which considering the authority Naval Officers have is quite immense.

While Central Command can clear this up IC and Administrators can reprimand someone for doing this, it doesn't curb the damage that the other player can/has do/done by wielding this authority for their own amusement and/or gains which can ruin a round for multiple players if not the entire player base playing at the time.

If players want to fantasize they're part of the NT Navy as these VIP roles, that's fine, we can't exactly stop them. However this is drawing a clear, tangible line that these occupations do not and never will wield the authority of Central Command, for new and old players alike. There'll be nothing in game that will allow players to point and go "Look, I'm a Naval Officer, you **HAVE** to do as I say."

**Comment by Carthusia** [comment]
> > I would still argue that they are Navy
> 
> Setting aside the lore aspects you're arguing, we're more concerned with the game aspects as well as the implications of such that occur with players.
> 
> Players even associating them as Naval Personnel is a problem, especially newer players, since Central Command Naval Officers wield absolute authority over everyone on station, including but not limited to ignoring, changing, altering, and dictating Space Law as well as Standard Operating Procedure.
> 
> If a player confuses one of the VIP as a Naval Officer, all it takes is the player playing the specific role to decide not to correct them and flout authority that they don't have, which considering the authority Naval Officers have is quite immense.
> 
**(While yes, a high ranking NTN officer does wield an INSANE amount of power, the potential for confusion seems quite low, considering the karma requirement for these positions ranges from 30 to 45, a player who has reached that amount can at the very least be expected to not attempt to go powermad, and if they do, a simple talk with and its resolved. As much as they may try to wield ultimate authority if they are confused, the bottom line is, they don't, and it will be made very clear to them,very fast should they try this.)**

> While Central Command can clear this up IC and Administrators can reprimand someone for doing this, it doesn't curb the damage that the other player can/has do/done by wielding this authority for their own amusement and/or gains which can ruin a round for multiple players if not the entire player base playing at the time.

**(Again with my above statement, the ability for say an NTR, to attempt to pull some kind of ultimate authority powerplay seems negligable, considering the inbuilt power checks to prevent someone from doing so. The chain of command is fairly clear, and the NTR is right near the bottom, UNTIL such time they are granted the authority to act.)**
> 
> If players want to fantasize they're part of the NT Navy as these VIP roles, that's fine, we can't exactly stop them. However this is drawing a clear, tangible line that these occupations do not and never will wield the authority of Central Command, for new and old players alike. There'll be nothing in game that will allow players to point and go "Look, I'm a Naval Officer, you **HAVE** to do as I say."

**If they are not officers, they are not really able to fantasize about anything, because a key part of their character is now gone. One could argue it was never there, though I would hazard a guess most people are in the belief that they are in fact low ranking officers.

Any attempt to do so would result in repremends from CC or other onboard staff, as they are in no way apart of the Officers corp, and are just bog standard civilians, who for some reason NT is trusting with possible corporate secrets, wetworks, and other horrifying bits of information that would see them lambasted by the outside world should anyone ever find out.

To me the clear, tangible line about how they don't wield the power of a military officer, on a military assignment, is that they by default do not hold that kind of power, they are a low ranking military officer, on a civilian assignment, and the players will make this clear if they try to muscle into things, just like a HoP attempting to supercede the captain, just like a detective attempting to supercede security, just like a QM trying to supercede the HoP.

The checks and balances for this kind of thing are already in place and in over 1000 hrs of playtime, i've not once seen an NTR,BS or Magistrate attempt some kind of excessive powerplay beyond their purview.
Maybe it happens, i don't deny this, but it seems to be a very small % of people who even attempt this kind of thing and are not almost immediately put into place.

To me, it makes logical sense for them to be low ranking naval officers and be on deployment to a civilian station, with limited authority to allow the station to function in a normal manner until such time corporate is informed of an incident, and further actions need to be taken. At which time, the cards fall where they may.
**

**Comment by Ty-Omaha** [comment]
like I said - argue all you want but it's not changing the fact that they aren't navy

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -123,11 +123,11 @@
 
 /obj/item/clothing/under/rank/centcom/blueshield/New()
 	..()
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant\" and bears [station_name()] on the left shoulder."
+	desc = "Gold trim on space-black cloth, this uniform bears \"Close Protection\" on the left shoulder."
```
Comment (Citinited) [inline code/modules/clothing/under/miscellaneous.dm]: This `New()` proc isn't required, now you aren't using a variable in the item's desc

**Comment by Carthusia** [comment]
> like I said - argue all you want but it's not changing the fact that they aren't navy

Sihsse made counter arguments that did have merit to them, though I don't fully agree with all of them but I at least see where they are coming from and can understand it a bit more now.

Your response isn't exactly as constructive as it appears you are simply ignoring the other side of a debate and this isn't exactly the way to go, but if that is how you wish to go about things, then do as you wish.

It just looks poorly on you.

**Comment by Carthusia** [comment]
I know full we'll i do not get to decide, I am merely stating my opinion on the subject.

Points 1-4 do not exactly prevent me from stating my opinion in a debate or prevent others from potentially seeing another side to an issue they may not have otherwise seen and vise versa.

A healthy open debate is always preferable with a multitude of perspectives.

**Comment by Sihsse** [comment]
A healthy and open debate requires one to be able to look at the other side. You swept my game play concerns under the rug essentially and didn't provide any instance where my concerns would be alleviated or unfounded.

This isn't a healthy, open debate. This is essentially us talking to a wall at this point.

**Comment by Abydos2** [comment]
> A healthy open debate is always preferable with a multitude of perspectives.

To quote yourself

> you are simply ignoring the other side of a debate and this isn't exactly the way to go

You've ignored every point that others or admins have made, you've ignored anything regarding confusion.. you even ignore what admins have seen from their long time of experience, and pulled a "Well I never seen it myself so you must be making it up". 

If you keep saying you want to debate, then completely ignore what the other said says while also pulling a "We agree to disagree" or "You're ignoring me, it looks poorly on you" when you yourself are ignoring them. Then their IS no debate. You are merely choosing to decide "I'm 100% right in everything, you're wrong, I choose to not believe you and make it out as if you're the problem".

To quote Winston Churchil

> Some people's idea of free speech is that they are free to say what they like, but if anyone says anything back, that is an outrage.

**Comment by Carthusia** [comment]
> A healthy and open debate requires one to be able to look at the other side. You swept my game play concerns under the rug essentially and didn't provide any instance where my concerns would be alleviated or unfounded.
> 
> This isn't a healthy, open debate. This is essentially us talking to a wall at this point.

I did not ignore your concerns? I simply made counter points that said concerns may already be mitigated ingame through the already in place mechanics such as the Chain of Command and listed SOP.

As for Abdyos2's reply: I have not ignored their points either, i've made counter points to them yes, but I have not ignored them.

In the context of the situation, my comment was a comparison of the helpfulness of two replies.

Sihsse's was concise and granted me a viewpoint that I had not considered, and thus formed my counter points to their points, both sides see where their coming from and can understand why they made the points they made and discuss until a middleground is found.

Ty-Omaha's just kind of seems like a "It's going to be this way no matter what" response and I in no way understand their point of view as to why beyond "This is the way it's going to be" and the discussion is shut down.

If Sihsse's response is in fact the official standing of the matter from an administrative viewpoint (I am unable to tell who is or isn't an administrator here so excuse me if the above is incorrect), then again as noted, I do see where your coming from, I do not fully agree with it, but I do see it .Do you see where i'm coming from, and why i've made the points i've made?

**Comment by Sihsse** [comment]
@Carthusia 
Let me approach this in a different way then.
Why do these three roles **HAVE** to be Navy?
NanoTrasen is first and foremost a Corporation, a very powerful one at that, and the NT Navy is their military arm so to speak. The company itself is vast with various personnel and the Navy doesn't encompass the entirety of the Company.

So while you say it makes sense for them to be Navy, I'm countering that it doesn't particularly since that the Navy while operating the Administrative Hub of Epislon Eridani is the ultimate authority in the sector involving NanoTrasen station assets, there are other sections of the Company no doubt incorporated and working in parallel with them that doesn't fall under the militaristic branch.

**Comment by Carthusia** [comment]
> @Carthusia
> Let me approach this in a different way then.
> Why do these three roles **HAVE** to be Navy?
> NanoTrasen is first and foremost a Corporation, a very powerful one at that, and the NT Navy is their military arm so to speak. The company itself is vast with various personnel and the Navy doesn't encompass the entirety of the Company.
> 
> So while you say it makes sense for them to be Navy, I'm countering that it doesn't particularly since that the Navy while operating the Administrative Hub of Epislon Eridani is the ultimate authority in the sector involving NanoTrasen station assets, there are other sections of the Company no doubt incorporated and working in parallel with them that doesn't fall under the militaristic branch.

My viewpoint on why a naval posting is required for something like say the NTR is just due to the possibility of highly secret corporate information being given to them, them being assigned to do something that otherwise would be illegal in any sector, things you absolutely do not want people to know about. The possibility of such a scenario to me, requires someone who can be wholly trusted to put corporate interests first. A civilian hired into the job just doesn't seem like something you can logicly trust with these possibilities as a MS implant isn't brainwashing the host, its merely hardening against your mind against being altered.
(Example: NTR being instructed by NTN S.O Officers to have a member of the crew killed for wanting to quit the company{Hotel California}) This is what has happened to me as an NTR, and is absolutely not something the general public should ever hear about due to internal issues it would cause within its own employment base, as well as external issues from surrounding states.

My viewpoint on why a naval posting is required for something like the BS is again, corporate loyalty, trust, and trained experience. While a CO posting to that of a bodyguard may be odd, that of a NCO like say, a chief petty officer, or a petty officer first class would fit quite nicely.

My viewpoint on the Magistrate on the other hand, is in line with your own. They are a member of the NTSC, the judicial branch of the company and their controlled sectors, not the military branches, and while loyalty, experience are still required, their area of expertise is less that of protection/combat and possible corporate secrets, than the open/public interpretation and application of NT Spacelaw.

The NTN has seemingly absolute authority within NT sectors, the other divisions, departments, almost seem irrelevant when compared to the NTN's level of power, like a military dictatorship. Even the supreme court seemingly has no authority over them, as the NTSC has final say in how spacelaw is applied, but the NTN can change it as they see fit, thus negating any authority the NTSC may have had over them.

That being said, while the NTN has seemingly absolute authority within the NT sectors, not every member of it would have the same level of authority. A first grade Ensign would be entrusted with a small level of authority due to the CO position while on military deployment, but in no way would hold the same level of mind bendingly high authority as say an Admiral or even a Navy Captain on military deployment to the Cyberiad.

To me: It makes sense for the NTR and BS to be members of the NTN for the above listed reasons, but while they are members of the NTN, they are in no way on a military deployment, they are deployed to a civilian research station for a very specific purpose and are not to deviate from that purpose unless authorized from their higher ups.

This is just my viewpoint on their positions from a lore perspective

**Comment by Abydos2** [comment]
If we go by your viewpoint and logic Carthusia, using the US Navy Ranking system as a bare basics example. You're saying the NTR is a Ensign, and the BS is a Lieutenant.

In the US System, that puts the BS above the NTR as an Ensign is 0-1 and a LT is 0-3. Yet if you look at the lore and everything else, the BS answers directly to the NTR. It even states in the Blueshield wiki it self:

> A Blueshield is a member of a specially trained branch of NanoTrasen employees

> The Nanotrasen Representative functions as an advisory role aboard the NSS Cyberiad. Important to note that the Nanotrasen Representative has no formal authority on the station whatsoever. The position of Nanotrasen Representative is only an advisory role and you are not permitted to directly intercede in the affairs of the Captain or the Heads of Staff

> The Magistrate reports to the Nanotrasen Supreme Court.

 You trying to force things from YOUR point of view does not make sense with just about anything you are saying, you are trying to enforce a system on something that has already been established as well as constantly go against multiple people, lore, facts and paperwork that says that they are not navy. If the Blueshield is a Navy Officer, why is he guarding people BELOW him?

If the NTR is a Navy Officer and trusted with company secrets why would;

1) NT send him to a research and mining station of all places

2) Both NT and CC make his position advisory only

3) Both NT and CC make it clear that he is a Civilian

If the Magistrate is a Navy Officer, why does he report to Nanotrasen Supreme Courts and not Central Command, or the Board of Directors?

You keep pointing out "My view point" "This is how I view it" "This is how I see" when people have written things down and you ignore them, pulling a "Well I don't agree with that, my view point is better and makes more sense." They are NOT navy officers, they are NOT part of CC and everything that points to them says they are not.. take the time to read the mission briefings and wiki on all of them. Nothing points to having any authority or power anywhere. Just because they have a uniform that says 1 description from years ago, does not mean they are navy.

**Comment by Carthusia** [comment]
I feel like you did not read my entire post Abdyos, as I noted the Blueshield conundrum above and noted a possible correction.

You also seem to have this idea im trying to force things onto people? I have no such intention nor even remotely that kind of authority to do so? I have merely made my viewpoint known and at this point responded in full to Sishsse's post.

But to respond to your points:

"If the NTR is a Navy Officer and trusted with company secrets why would;

1. NT send him to a research and mining station of all places
**(Considering how valuable the station is, and that NT is seemingly a Military Corporatocracy, coupled with their low rank, there are a number of reasons why a low ranking officer would be assigned to the station as an advisor/overseer)**

2. Both NT and CC make his position advisory only
**(See above: Depending on the RP background of the character in the role, there are any number of reasons they are assigned to the station, from making someone mad, having volunteered for the assignment, or just being unlucky and being the only one who didn't opt out in time.** 
_(Anyone who wishes for this position, step forward: Everyone but you takes one step back: Right, Jones your assigned to the NSS Cyberiad)_

3. Both NT and CC make it clear that he is a Civilian
**(In this case, a military officer, operating onboard a civilian station, placed into a civilian administrative role, is still a civilian as they hold limited authority and are bound by Civilian Space Law, but is also still an officer. It is dependent entirely on the deployment parameters that are set out for them. Again: See above: Any number of reasons why they have had these parameters enforced, which is kind of up to players to create backgrounds for characters as to why this is the case.)**

"Just because they have a uniform that says 1 description from years ago, does not mean they are navy."
**(This uniform description seems to have permeated into the very being of the character roles due to having been there for so long it would seem, as I see people refer to the BS as Lt. and salute both him and the NTR on a regular basis, but I never see them confused for, or ever being thought to have the same authority or power as a Navy Captain or Admiral)**

**Whether you agree with my viewpoint or not is entirely up to you, I am just making it known, and if someone agrees with me, that's fine, and if no one does, that's also fine. I don't see the need for such an aggressive tone while putting words in my mouth. You can simply disagree.**

**Comment by Abydos2** [comment]
It seems to be that you are also ignoring the whole post, given that you only seem to focus on the parts that you focus on and not the whole picture. Since you keep focusing more on how you claim that everyone is ignoring you, let's just break your whole post down:

> NT is seemingly a Military Corporatocracy

Where is this stated anywhere within lore? Where does it say they are a Military Corp? They have a Navy as part of their defenses, that does not mean they are Military. If you read the lore, you'd see and I quote:

> The largest shareholder in the galactic plasma markets, Nanotrasen is a research and mining corporation which specializes in FTL technologies and weapon systems. Frowned upon by most governments due to their shady business tactics and poor ethics record, Nanotrasen is often seen as a necessary evil for maintaining access to the volatile plasma market. Though Nanotrasen was originally incorporated on Earth with their headquarters situated on Mars, they have relocated many of their research operations to the Epsilon Eridani sector. Nanotrasen is the largest corporation in the galaxy, its power rivaling most governments. It deals in research of the most advanced sciences, such as genetics, blue space, and - recently - the uses of plasma. They also have a sizeable asset protection and security branch with which they secure both investments and occasionally new acquisitions - a distressing rival to some governments, and a powerful administrative branch sitting atop it all, directing their company's actions.

> there are a number of reasons why a low ranking officer would be assigned to the station as an advisor/overseer

An overseer means they have weight and in charge, aka, the Captain. The Captain's not Navy.. but the NTR is Navy according to your logic, meaning, you're trying to say the NTR has more authority than the Captain assigned to RUN and OVERSEE the station. What's the point of having a Captain then if the NTR is Navy and has more weight than the guy specifically picked to run the station?

> Depending on the RP background

No, there is no "Depending on the RP background" because giving that sort of freedom to just about anyone, means, you are opening up the gates to allow everyone to claim they are CC officers because their "Background" lets them. Doing the very thing that the admins have both punished and put a stop too constantly.

> , a military officer, operating onboard a civilian station, placed into a civilian administrative role, is still a civilian as they hold limited authority and are bound by Civilian Space Law, but is also still an officer. 

That is literally a double standard. The guy who's above them is still above them but when he's in an area he's still a civilian until a specific incident rises up. That makes absolutely 0 sense.. what's the point of sending him in the first place then if he's still, considered, for all intents and purposes, a Navy officer? That's the same thing as above regarding the "RP backgrounds". 

> I see people refer to the BS as Lt. and salute both him and the NTR on a regular basis

I see people refer to the Captain as the Captain and salute him. The ranks that are printed on their uniforms hold no meaning, we've already established that the NT Rep is a company man outside of CC / Navy. The Magistrate, law guy employed by the Supreme Courts. The Blueshield, one of handpicked employees to be a Bodyguard on a specific branch. None of them are Navy, all the lore points that out, that you keep ignoring and going "They're navy in my point of view, look, they got a uniform that says Ensign on it."

> Whether you agree with my viewpoint or not is entirely up to you

Admins have been telling you that your viewpoint is wrong and not agreed upon.. to which you took to attacking one of them claiming they are ignoring you and not debating you, when you had previously stated "We agree to disagree and **leave it at that"** as if we cannot do the PR without your approval. Multiple admins have made it clear that this needs fixing, this PR is aimed to fix it.. and you seem to be very aggressive at trying to establish and enforce that the NTR, the BS and the Magistrate are Navy officers. They are not.. 

> You can simply disagree.

You could simply agree with the written lore, multiple admin comments and the facts as they are presented that they are not, you are choosing not too and trying to enforce your view point against everything.

**Comment by Carthusia** [comment]
@Sihsse If you wish to discuss the lore side of things on Discord by all means contact me, I do enjoy lore talks and seeing other viewpoints on it.

Alot of your questions @Abydos2 are answered within my responses prior. My invitation to discuss it more in depth is also available to you should you wish to take me up on it.

As for here, there's not much more I can say, if we continued this would become a Lore discussion/debate area. You may not fully agree with my viewpoint on this, and I may not fully agree with yours. 

What will be, will be, and people will accept it and move on. My purpose in this was to bring a different viewpoint from an angle some may not have considered prior.

**Comment by Ty-Omaha** [comment]
also ready for merge with #12706 @variableundefined

**Comment by C-Kuha** [comment]
What a mess. A disappointing mess too.

**Pull request merged**